jupyter-notebooks/orders_basemaps/requests_order_basemaps.ipynb

```diff
@@ -0,0 +1,654 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def poll_for_success(order_url):\n",
+    "    '''Polls for order success, fails after num_loops\n",
+    "\n",
+    "    Parameters: order_url (str): url for order\n",
+    "                num_loops (int): number of times to poll before failing\n",
+    "    Prints the state of the order\n",
+    "    '''\n",
+    "    state = ''\n",
+    "    end_states = ['success', 'failed', 'partial']\n",
+    "    while state not in end_states:\n",
+    "        r = requests.get(order_url, auth=session.auth)\n",
+    "        response = r.json()\n",
+    "        state = response['state']\n",
+    "        print(state)\n",
+    "        if state in end_states:\n",
+    "            break\n",
+    "        time.sleep(10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "queued\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "success\n"
+     ]
+    }
+   ],
+   "source": [
+    "poll_for_success(order_url)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "basemaps-to-cloud/31c7dd26-c8a0-4897-a56f-83e1ef0cf40f/global_monthly_2022_01_mosaic/global_monthly_2022_01_mosaic_provenance_raster_merge_clip.tif\n",
+      "basemaps-to-cloud/31c7dd26-c8a0-4897-a56f-83e1ef0cf40f/global_monthly_2022_01_mosaic/global_monthly_2022_01_mosaic_merge_clip.tif\n",
+      "basemaps-to-cloud/31c7dd26-c8a0-4897-a56f-83e1ef0cf40f/global_monthly_2022_01_mosaic/global_monthly_2022_01_mosaic_provenance_vector_merge_clip.zip\n",
+      "basemaps-to-cloud/31c7dd26-c8a0-4897-a56f-83e1ef0cf40f/global_monthly_2022_01_mosaic/global_monthly_2022_01_mosaic_metadata_merge_clip.json\n",
+      "basemaps-to-cloud/31c7dd26-c8a0-4897-a56f-83e1ef0cf40f/global_monthly_2022_01_mosaic/global_monthly_2022_01_mosaic_ortho_udm_merge_clip.tif\n",
+      "basemaps-to-cloud/31c7dd26-c8a0-4897-a56f-83e1ef0cf40f/manifest.json\n"
+     ]
+    }
+   ],
+   "source": [
+    "r = requests.get(order_url, auth=session.auth)\n",
```

Comment:
You could just have `poll_for_success` return the last response, instead of making another request here, since that will contain the download links already.